includes/definitions/discovery/moxa-etherdevice.yaml

```diff
@@ -0,0 +1,16 @@
+modules:
+    sensors:
+        power:
+            data:
+                -
+                    oid: MOXA-IKS6726A-MIB:powerConsumption
+                    num_oid: .1.3.6.1.4.1.8691.7.116.1.10.8.
+                    descr: 'Power Consumption'
+                    index: 'powerConsumption.{{ $index }}'
+                    value: 'powerConsumption'
```

Comment:
You can drop the quotes from this, so it's just `value: powerConsumption`. Not strictly needed but all others are like that.